src/interfaces.d.ts

```diff
@@ -1,21 +1,50 @@
-export interface Column<T> {
+import { DNode } from '@dojo/widget-core/interfaces';
+
+/**
+ * @type Column
+ *
+ * Adds information about a column that will be used by the
+ * grid when retrieving and displaying data.
+ *
+ * @typeparam T        Used by field property and cell customization functions to enforce type safety
+ * @typeparam V        Used by cell customization to ensure the same type is used by get and render
+ * @property  field    Property on the row's data item to use to look up the column's value
+ * @property  id       A unique identifier for this column
+ * @property  label    The label to use in the column's header
+ * @property  sortable Set to false to indicate the column is not sortable
+ * @property  get      Manually return the value or content to use for this column
+ * @property  render   Use this column's value to provide content for this column
+ */
+export interface Column<T = any, V = string> {
```

Comment:
yes, we need to make it explicit, we have been trying to alphabetize things by default, as when they get added to, it becomes difficult to find stuff in big interefaces... properties then methods, yup, but alphabetical within those groups